week-3/A-array-find/exercise.js

```diff
@@ -6,7 +6,11 @@
 // write your code here
 
 var names = ["Rakesh", "Alexandra", "Annam", "Mikey", "Karim", "Ahmed"];
-var longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+var longNameThatStartsWithA = names.find(longNameThatStartsWithA)
+
+function longNameThatStartsWithA(name) {
+    return name[0] === "A" && name.length > 7;
```

Comment:
This is correct, but we can make a small improvement. What happens if we change the `names` array so that it has "alexandra" instead of "Alexandra"? Does it still work? How could we fix it?

Reply:
@40thieves Thanks for pointing that out, I could improve the code by just returning name[0] ===name[0] && name.length >7